public/docs/ts/latest/guide/reactive-forms.jade

```diff
@@ -0,0 +1,885 @@
+
+:marked
+  Here’s a summary description of the imported symbols, whose meaning and use
+  will become clear over the course of this guide:
+
+  - `Component`: the Angular decorator that describes the component in metadata.
+  - `EventEmitter`: a class that emits events that are handled by a parent component.
+  - `Input`: identifies a component's public input properties for data binding by a parent component.
+  - `Output`: identifies a component's public output properties for data binding by a parent component.  
+  - `OnChanges`: a component lifecycle hook that Angular calls when input values change.
+  - `FormArray`: an array of `FormControls`.
+  - `FormBuilder`: a builder of `FormControls`, `FormGroups`, and `FormArrays`.
+  - `FormControl`: a representation of an HTML form control.
+  - `FormGroup`: a group of `FormControls`, `FormArrays`, and other `FormGroups`.
+  - `Address` and `Hero` are application model classes.
+  - `states`: U.S. states (test data). 
+
+  Now enter the `@Component` decorator that specifies the `HeroDetailComponent` metadata:
+
++makeExample('reactive-forms/ts/app/hero-detail.component.ts', 'reactive-comp-metadata','app/hero-detail.component.ts (excerpt)')(format=".")
+
+:marked
+  The `moduleId: module.id` lets you use 
+  [component-relative paths](./cookbook/component-relative-paths.html) in file URLs. 
+  The `selector` tells Angular the name of the element tag that renders this form. 
+  The `templateUrl` is the location of the HTML template that displays the form.
+
+  Next, create an exported `HeroDetailComponent` class with a `FormControl`. 
+
++makeExample('reactive-forms/ts/app/hero-detail-versions.component.ts', 'v1','app/hero-detail.component.ts (excerpt)')(format=".")
+
+:marked
+  Inside the `FormGroup`, called `heroForm`, is a `FormControl` called `name`.
+  It will be bound in the template to an HTML `input` box for the hero name. 
+
+  A `FormControl` constructor accepts three, optional arguments: 
+  the initial data value, an array of validators, and an array of async validators.
+  
+  This simple control doesn't have data or validators.
+  In real apps, most form controls have both. 
+
+.l-sub-section
+  :marked
+    Validators won't be covered at all in this guide.
+    Learn about them in the [Form Validation](../cookbook/form-validation.html) cookbook.
+
+.l-main-section
+a#create-template
+:marked
+  ## Create the template
+  Now create the component's template, `app/hero-detail.component.html`, with the following markup.
+
++makeExample('reactive-forms/ts/app/hero-detail-1.component.html', 'basic-form','app/hero-detail.component.html')(format=".")
+
+:marked
+  The `novalidate` attribute in the `<form>` element prevents the browser 
+  from attempting native HTML validations. 
+  The `[formGroup]="heroForm"` tells Angular that the form element
+  is associated with the `heroForm` property of the component class.
```

Comment:
I think you're conflating `FormGroup` and `formGroup`, which is why I'd like to add this clarification.  `FormGroup` is a class that is shared between both modules. `formGroup` (a.k.a. FormGroupDirective) is a reactive form directive that exists *only* in the ReactiveFormsModule.